INSTALL

```diff
@@ -1,16 +1,20 @@
 Installing Cartopy
 ==================
 
-Pre-built binaries
-------------------
+Conda pre-built binaries
+------------------------
 
-The easiest route to installing cartopy is through 
-`Conda <https://conda.io/miniconda.html>`_. For all platforms
-installing cartopy can be done with::
+The easiest way to install Cartopy is by using
+`Conda <https://conda.io/miniconda.html>`_. If conda is already installed,
+installation is as easy as::
 
     conda install -c conda-forge cartopy
 
-Additional options include:
+
+Other pre-built binaries
+------------------------
+
+Additional pre-built binaries can be found at a variety of sources, including:
  * `Enthought Canopy <https://www.enthought.com/product/canopy/>`_.
```

Comment:
I couldn't find a pre-built binaries here:    `Enthought Canopy <https://www.enthought.com/product/canopy/>`

Reply:
Looks like Enthought re-branded or something. They now have the [Enthought Deployment Manager](http://docs.enthought.com/edm/), but I can't find anything on the binaries available.

Reply:
I just removed the link to Enthought.